config/jest/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Be good to go through after and remove the duplicated adapters in the other tests files